std_msgs/Header header
geometry_msgs/Point point
string class_name
